<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureToolPanel</class>
 <widget class="QWidget" name="MeasureToolPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeasureToolPanel</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pBtnArea">
   <property name="geometry">
    <rect>
     <x>1931</x>
     <y>10</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>测量面积</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnLine">
   <property name="geometry">
    <rect>
     <x>193</x>
     <y>10</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>测量直线距离</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnAngle">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>10</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>测量角度</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnPoint">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>10</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>测量直线距离</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnPlane">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>10</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>测量直线距离</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PBtnOpenCurve">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>10</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>测量直线距离</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>1901</x>
     <y>40</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>面积</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>角度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>点</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>平面</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>曲线</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PBtnClose">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>测量直线距离</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>477</x>
     <y>40</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>隐藏</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
